src/disco/containers/DiscoPane.js

```diff
@@ -20,6 +20,21 @@ class DiscoPane extends React.Component {
     const { results } = this.props;
     return (
       <div id="app-view" ref="container">
+        <header>
+          <div className="content">
+            <h1>{_('Personalize Your Firefox')}</h1>
+            <p>{_(`There are thousands of add-ons that let you make Firefox all your
+               own—everything from fun visual themes to powerful tools and features.
+               Here are a few great ones to check out.`)}</p>
+          </div>
+          <a href="#" className="play-video">
+            <p>
+              <span>{_('Click to play')}</span>
+              <span className="visually-hidden"> _('to find out more about add-ons')</span>
```

Comment:
This is missing the `{}` so I think it's actually writing out the `_('')` bit to the page.